homeassistant/components/icloud/account.py

```diff
@@ -104,6 +104,8 @@ def __init__(
         self._retried_fetch = False
         self._config_entry = config_entry
 
+        self.listeners = []
```

Comment:
Indeed I removed it by mistake. I thought it was a leftover from a previous refactor.

Could you add the correct type hint to fix mypy?
```suggestion
        self.listeners: list[xxx]= []
```

Reply:
Note: I have created an alternative PR #76437 which adds a couple of extra type hints (in device_tracker and sensor) to avoid the mistake in the future